### PR TITLE
Add description to community, and other fixes

### PR DESCRIPTION
**Pull request by cwant**
This PR is part of issue #38. The community logo will be a separate PR.

Also within this PR:

* Add validation for presence of community title
* Make community#show view more closely resemble the mockup:
  https://jupiter.mybalsamiq.com/projects/jupiter/Admin%20-%20community%20detail%20page
* Fix for incorrectly-nested translations (also use lazy lookup)

### CODE REVIEW

File: app/views/communities/show.html.erb [hunk 1]
```diff
@@ -1,25 +1,52 @@
...
-        <% end %>
+        <%= t('.collections_list_header') %>
       <% else %>
         <%= t('.no_collections') %>
       <% end %>
-    </div>
+    </h4>
+  </div>
+  <div class="col text-right">
+    <%= link_to t('.create_new_collection'), new_community_collection_path(@community),
```
Comment (murny) [inline app/views/communities/show.html.erb]: Wrap this with an authorize check? I assume only admins should get to see the new collection button?

File: test/integration/community_show_test.rb [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+class CommunityShowTest < ActionDispatch::IntegrationTest
+
+  setup do
+    admin = users(:admin_user)
+
+    # TODO: setup proper fixtures for LockedLdpObjects
+
+    # A community with two collections
+    @community1 = Community.new_locked_ldp_object(title: 'Two collection community',
```
Comment (mbarnett) [inline test/integration/community_show_test.rb]: Take a look at https://github.com/ualbertalib/jupiter/pull/82 and refactor this all into an `before_all` method. `setup` runs before every single test, so this wastes a lot of time with unnecessary writes.

File: test/integration/community_show_test.rb [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+    # A community with no collections
+    @community2 = Community.new_locked_ldp_object(title: 'Empty community',
+                                                  owner: admin.id)
+    @community2.unlock_and_fetch_ldp_object(&:save!)
+  end
+
+  teardown do
+    ActiveFedora::Cleaner.clean!
+  end
```
Comment (mbarnett) [inline test/integration/community_show_test.rb]: Once you've moved all the creates into a `before_all`, you can delete the `teardown`, as Fedora/Solr are already cleaned between each file.

File: test/integration/community_show_test.rb (hunk 2 above)
Comment (murny) [inline test/integration/community_show_test.rb]: Another option here is if you want to setup some stuff for only certain tests, you can nest those certain tests and the setup block under a `context` block. This way only things in the context block are affected and not everything else. 

But in this case probably matt's suggestion is better

Comment (cwant) [inline test/integration/community_show_test.rb]: Yeah, I had originally set it up to use a context block, but for some reason the tests were complaining about "missing argument v" or such.

Anyways, will explore the options.

Comment (cwant) [inline test/integration/community_show_test.rb]: Tests updated -- thanks!

**Pull request merged**